command/issues/list/list.go

```diff
@@ -46,19 +47,20 @@ type Summary struct {
 func NewCmdIssuesList() *cobra.Command {
 
 	opts := IssuesListOptions{
-		FileArg:  "",
+		FileArg:  []string{""},
 		RepoArg:  "",
 		LimitArg: 30,
 	}
 
 	cmd := &cobra.Command{
 		Use:   "list",
 		Short: "List issues reported by DeepSource",
-		Args:  utils.MaxNArgs(1),
+		Args:  utils.MaxNArgs(15),
```

Comment:
What number should we use here?

Reply:
I guess since the default behaviour of `deepsource issues list` without any argument is to list all the issues from the default analysis branch for the whole repo thus, it would even work without any argument. Thus, I guess we can remove argument validation for this command. What do you think @burntcarrot ?

Reply:
Cool